docs/developer/adding-community-components.md

```diff
@@ -0,0 +1,53 @@
+# Adding community components
```

Comment:
Can we have a section briefly describing when something should be a community component vs an official component? 

I think mainly it's about vendors, right?